sympy/series/tests/test_order.py

```diff
@@ -157,15 +157,15 @@ def test_add_1():
     assert Order(3*x - 2*x**2) == Order(x)
     assert Order(1 + x) == Order(1, x)
     assert Order(1 + 1/x) == Order(1/x)
-    assert Order(log(x) + 1/log(x)) == Order(log(x))
+    assert Order(log(x) + 1/log(x)) == Order((log(x)**2 + 1)/log(x))
```

Comment:
This can probably be fixed later in order.py.

Reply:
Yeah I remember asking this on the gitter channel , if we could do with this ans . Not sure if I got any suggestion on that . Would this do ? or we should stick to `log(x)` as before . In any case I'll give it a quick try regarding how this can be fixed , otherwise make a TODO for that !

Reply:
Ahh yeah I remember , this comes out of this line we had added
```
        _logx = Dummy('logx') if logx is None else logx
        leading_terms = [t.as_leading_term(x, logx=_logx, cdir=cdir) for t in expr.args]
```
Treating log as constants means log(x) and 1/log(x) both are rather treated as constants/symbols , hence the second log(x) term isn't eliminated !